Assets/Scripts/Core/Portal.cs

```diff
@@ -274,7 +273,7 @@ public class Portal : MonoBehaviour {
     void OnTravellerEnterPortal (PortalTraveller traveller) {
         if (!trackedTravellers.Contains (traveller)) {
             traveller.EnterPortalThreshold ();
-            traveller.previousOffsetFromPortal = traveller.transform.position - transform.position;
+            portalSideOld = SideOfPortal(traveller.transform.position);
```

Comment:
Instead of setting the previous offset from the portal, save the side of the portal on entry. This removes the issue caused by the lag between LateUpdate and FixedUpdate